frontend/app/src/components/helper/Card.vue

```diff
@@ -26,27 +26,21 @@ const props = withDefaults(
 );
 
 const { contained } = toRefs(props);
-const body = ref<HTMLDivElement | null>(null);
-const actions = ref<HTMLDivElement | null>(null);
-const top = ref(206);
+const titleRef = ref<HTMLDivElement | null>(null);
+const subTitleRef = ref<HTMLDivElement | null>(null);
+const actionsRef = ref<HTMLDivElement | null>(null);
 
-onMounted(() => {
-  setTimeout(() => {
-    set(top, get(body)?.getBoundingClientRect().top ?? 0);
-  }, 1000);
-});
-
-const bodyStyle = computed(() => {
-  if (!get(contained)) {
-    return null;
-  }
-  const bodyTop = get(top);
-  const actionsHeight = get(actions)?.getBoundingClientRect().height ?? 0;
-  const diff = bodyTop + actionsHeight;
+const { height: titleHeight } = useElementBounding(titleRef);
+const { height: subTitleHeight } = useElementBounding(subTitleRef);
+const { height: actionsHeight } = useElementBounding(actionsRef);
 
-  return {
-    height: `calc(100vh - ${diff}px)`
-  };
+const otherHeights = computed(() => {
+  const subTitleHeightVal = get(subTitleHeight) ?? 0;
+  const totalHeight =
+    (get(titleHeight) ?? 0) +
+    (subTitleHeightVal > 0 ? subTitleHeightVal - 16 : subTitleHeightVal) +
```

Comment:
Is 16 the padding?

Reply:
> Is 16 the padding?

So -16 is margin-top of the subtitle (if exist), so need to be substracted

Reply:
> if so do you think we can use `getComputedStyle` on the element to dynamically calculate it?

yes let me try